src/prefect/environments/storage/docker.py

```diff
@@ -79,19 +72,50 @@ def __init__(
         self.image_name = image_name
         self.image_tag = image_tag
         self.python_dependencies = python_dependencies or []
+        self.python_dependencies.append("wheel")
+
         self.env_vars = env_vars or {}
+        self.env_vars["PREFECT__USER_CONFIG_PATH"] = "/root/.prefect/config.toml"
```

Comment:
Just out of curiosity why was this necessary?  I believe this should be the default.

Reply:
Nvm I now see that you just moved this from where it previously was 👍